examples/sample-project/sample-project-tests.postman_collection.json

```diff
@@ -26,6 +26,26 @@
 		{
 			"name": "Get layer 0 descriptor",
 			"request": {
+				"auth": {
```

Comment:
It of course is weird that a "writer" user is needed to "Get a layer", but of course that's how it works today. I'd rather do this than add the "writer" role to the "reader" user.